By default rk808's buck regulators switch voltage in multiple 100mV jumps.
This seems to be too much for NanoPi M4V2 and makes it unstable.

Shortening the steps to 50mV (4 multiple of base buck step which is 12.5mV)
makes the NanoPi M4V2 stable.
Tested with multiple hours of running memtester without a single failure.

Signed-off-by: Piotr Szczepanik <piter75@gmail.com>

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-nanopi-m4v2.dts b/arch/arm64/boot/dts/rockchip/rk3399-nanopi-m4v2.dts
index 2dcaf497c..094440ce3 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-nanopi-m4v2.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-nanopi-m4v2.dts
@@ -64,6 +64,10 @@ &gmac {
 	rx_delay = <0x16>;
 };
 
+&rk808 {
+	max-buck-steps-per-change = <4>;
+};
+
 &vcc3v3_sys {
 	vin-supply = <&vcc5v0_core>;
 };
@@ -80,3 +84,4 @@ &vbus_typec {
 	regulator-always-on;
 	vin-supply = <&vdd_5v>;
 };
+
